AY2122S1-CS2103T-T11-2/tp
# Create download and upload features

### Description

Created the Download and Upload features for the app.

### Type of change


### How to test

Manual testing:
1. Use download command by typing "download FILE_NAME" (e.g. "download test.csv")
2. Check if the CSV file has been created properly in the ./data directory
3. Use upload command by typing "upload FILE_PATH" (e.g. "upload ./data/test.csv")
4. Check if the cards in the CSV file are correctly added to the list of existing cards.

### Checklist
